index.bs

```diff
@@ -297,8 +318,25 @@ spec: HEADER-STRUCTURE; urlPrefix: https://httpwg.org/http-extensions/draft-ietf
       then return true.</li>
       <li>Otherwise, for each <var>item</var> in the <a>allowlist</a>:
         <ol>
-          <li>If <var>item</var> is [=same origin-domain=] with
-          <var>origin</var>, then return true.</li>
+          <li>If <var>item</var> is a tuple of a <a>serialized-origin</a> that's [=same origin-domain=] 
```

Comment:
You probably just want "an [=origin=]" rather than a serialized-origin, since allowlists are now defined to contain (origin,bool) tuples (same on line 323)